README.md

```diff
@@ -68,7 +68,7 @@ $string = str('john pinkerton');
 ```
 
 Once you have a concrete `Twine\Str` instance you may treat it like any other
-string. This includes echoing it or using any of PHP's built in string functions
+string. This includes echoing it or using any of PHP's built-in string functions
```

Comment:
"[built-in](https://www.merriam-webster.com/dictionary/built-in)" is correct.

Reply:
Yes, and that's what I used. Also, "built-in" is already used on line 82.

Reply:
Oh sorry, I read that backwards for some reason, thought you were removing the dash (maybe I shouldn't review PRs in the middle of the night).

Reply:
😄 No worries. Get some sleep!